lib/emque/producing/publisher/rabbitmq.rb

```diff
@@ -5,6 +5,9 @@ module Emque
   module Producing
     module Publisher
       class RabbitMq < Emque::Producing::Publisher::Base
+        Emque::Producing::Message::PUBLISH_IGNORED_EXCEPTIONS << Bunny::Exception
+        Emque::Producing::Message::PUBLISH_IGNORED_EXCEPTIONS << Timeout::Error
```

Comment:
It was getting more difficult to wrap the entire class in a rescue. Bunny.new throws exceptions and it was awkward to rescue that exception and convert it to Emque exception. This way each Publisher can specify which exceptions to ignore.